engine/events/hidden_objects/safari_game.asm

```diff
@@ -49,7 +49,7 @@ SafariZoneGameOver:
 	ldh [hWarpDestinationMap], a
 	ld a, $3
 	ld [wDestinationWarpID], a
-	ld a, $5
+	ld a, $5 ; SCRIPT_SAFARIZONEGATE_LEAVING_SAFARI
```

Comment:
Tried to set this to `ld a, SCRIPT_SAFARIZONEGATE_LEAVING_SAFARI`, but I get:  
\`\`\`  
error: main.asm(135) -> engine/events/hidden\_objects/safari\_game.asm(52): Unknown symbol "SCRIPT\_SAFARIZONEGATE\_LEAVING\_SAFARI"  
\`\`\`

Reply:
This can be resolved by merging or `INCLUDE`'ing `maps.asm` into `main.asm`. My recommendation is just to merge them and line up the sections in the appropriate order based on the `layout.link` file for consistency. We can either fix this in this PR; or we can delay it for a future PR and leave the comment of the constant here for now.

Reply:
`EXPORT SCRIPT_SAFARIZONEGATE_LEAVING_SAFARI`?

Reply:
> `EXPORT SCRIPT_SAFARIZONEGATE_LEAVING_SAFARI`?

For some reason I thought this had another purpose and skipped over it... re-read the rgbasm docs... duh.. that's what I needed. Thanks. I knew I was missing something.